### PR TITLE
 [Enhancement] (nereids)implement adminCheckTabletsCommand in nereids…

### LINKED ISSUE #42843: [Enhancement] (nereids)implement adminCheckTabletsCommand in nereids
adminCheckTablets command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following adminCheckTablets command in nereids:
unsupportedadminCheckTabletsStatement
``` ADMIN CHECK tabletList properties=propertyClause?                             ```

### PR DESCRIPTION
**Pull request by rijeshkp**
… #42843

### What problem does this PR solve?

Issue Number: close #42843

Related PR: #

Problem Summary:
Support ADMIN CHECK tabletlist command in nereids

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by rijeshkp** [comment]
run buildall

**Comment by rijeshkp** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import org.apache.commons.collections.CollectionUtils;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * drop user command
+ */
+public class AdminCheckTabletsCommand extends Command implements Redirect {
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java:41]:
```suggestion
public class AdminCheckTabletsCommand extends Command implements ForwardNoSync {
```
implements ForwardNoSync is better, you can remove toRedirectStatus

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        }
+    }
+
+    @Override
+    public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
+        return visitor.visitAdminCheckTabletsCommand(this, context);
+    }
+
+    @Override
+    public RedirectStatus toRedirectStatus() {
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java:102]:
```suggestion
```
no need to override toRedirectStatus if implements ForwardNoSync

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        if (Objects.requireNonNull(checkType) == CheckType.CONSISTENCY) {
+            Env.getCurrentEnv().getConsistencyChecker().addTabletsToCheck(tabletIds);
+        }
+    }
+
+    @Override
+    public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
+        return visitor.visitAdminCheckTabletsCommand(this, context);
+    }
+
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java:100]: override checkSupportedInCloudMode method and implement as checkDdlStmtSupported() method in DdlExecutor.java like bellow
```
@Override
protected void checkSupportedInCloudMode(ConnectContext ctx) throws DdlException {
    LOG.info("AdminCheckTabletsCommand not supported in cloud mode");
    throw new DdlException("Unsupported operation");
}
```

**Comment by rijeshkp** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java (hunk 3 above)
Comment (rijeshkp) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java:104]: Resolved. Please review

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java (hunk 2 above)
Comment (rijeshkp) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCheckTabletsCommand.java]: Resolved. Please review

**Review by starocean999 — APPROVED** [review]

**Review by LiBinfeng-01 — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**